### PR TITLE
Quiet the log line for "Could not find a matching ingredient to remove ..."

### PR DESCRIPTION
**Pull request by Goldenlion5648**
changed the spam to only happen on debug mode

### CODE REVIEW

**Comment by mezz** [comment]
Thanks for contributing!
Could you just make this a debug-level log instead?

**Comment by Goldenlion5648** [comment]
Done.

**Comment by mezz** [comment]
Ah sorry I wasn't very clear, I mean it doesn't even need to be locked behind `if(clientConfig.isDebugModeEnabled())` if it's just a debug log.
Can you just make it a 1-word change from `error` to `debug`? Thanks again :D

**Comment by mezz** [comment]
Actually I have an idea, I can edit it on github, one sec.

File: Forge/src/main/java/mezz/jei/ingredients/IngredientManager.java [hunk 1]
```diff
@@ -164,8 +164,10 @@ public <V> void removeIngredientsAtRuntime(IIngredientType<V> ingredientType, Co
 			.forEach(typedIngredient -> {
 				Optional<IListElementInfo<V>> matchingElementInfo = ingredientFilter.searchForMatchingElement(ingredientHelper, typedIngredient);
 				if (matchingElementInfo.isEmpty()) {
-					String errorInfo = ingredientHelper.getErrorInfo(typedIngredient.getIngredient());
-					LOGGER.error("Could not find a matching ingredient to remove: {}", errorInfo);
+					if(clientConfig.isDebugModeEnabled()) {
```
Comment (mezz) [inline Forge/src/main/java/mezz/jei/ingredients/IngredientManager.java]:
```suggestion
					if (LOGGER.isDebugEnabled()) {
```

**Review by mezz — CHANGES_REQUESTED** [review]

**Review by mezz — APPROVED** [review]

**Pull request merged**